includes/utils.sh

```diff
@@ -256,15 +256,15 @@ then
 		if [ "${RET0}" != "" ]
 		then
 			RET2=`eval norminette $RET0 2>&1`
+			RET4=$?
 			echo "$RET2" > "$RETURNPATH"/.mynorminette
 			RET2=`cat .mynorminette | grep Error`
 			RET3=`cat .mynorminette | grep Warning`
-			RET4=`cat .mynorminette | grep "command not found"`
 			if [ "$RET2" == "" -a "$RET3" == "" -a "$RET4" == "" ]
```

Comment:
EDIT: I just tested the two following sequences and it results in the same output :-)

```bash
RET4=`eval exit 42`
echo $?
``` 

```bash
`eval exit 42`
echo $?
``` 

You should replace the line 263 with:
```bash
if [ "$RET2" == "" -a "$RET3" == "" -a "$RET4" == 0 ]
```